.github/workflows/release.yml

```diff
@@ -13,8 +13,8 @@ jobs:
     strategy:
       matrix:
         include:
-          - runner: ubuntu-latest
-            cibw_arch: aarch64
+#          - runner: ubuntu-latest
+#            cibw_arch: aarch64
```

Comment:
keeping this commented because it's temporary and we want the visual reminder?

Reply:
yup, those are coming back, just wanted to reduce the cycle time on testing the windows specific portion of the build.